javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,5 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  const ch = string.charAt(index);
```

Comment:
You can do this, but you can also use brackets like an array to get the value! So you can do `string[index]`. Either way is totally fine, I just wanted to show you another way of doing this!

Reply:
got it, thank you!